src/cfdcore_schnorrsig.cpp

```diff
@@ -312,6 +314,54 @@ Pubkey SchnorrUtil::ComputeSigPoint(
   return ConvertSecpPubkey(secp_sigpoint);
 }
 
+Pubkey SchnorrUtil::ComputeSigPointBatch(
+    const std::vector<ByteData256> &msg,
```

Comment:
I looked at the header file, but I think the variable name here is msgs.
```suggestion
    const std::vector<ByteData256> &msgs,
```

Reply:
Yes had forgotten to update. Fixed.